src/shell/commands/table_management.cpp

```diff
@@ -886,3 +888,105 @@ bool clear_app_envs(command_executor *e, shell_context *sc, arguments args)
     }
     return true;
 }
+
+bool get_max_replica_count(command_executor *e, shell_context *sc, arguments args)
+{
+    if (args.argc < 2) {
+        return false;
+    }
+
+    std::string app_name(args.argv[1]);
+
+    auto err_resp = sc->ddl_client->get_max_replica_count(app_name);
+    auto err = err_resp.get_error();
+    const auto &resp = err_resp.get_value();
+
+    if (err.is_ok()) {
+        err = dsn::error_s::make(resp.err);
+    }
+
+    std::string escaped_app_name(pegasus::utils::c_escape_string(app_name));
+    if (err.is_ok()) {
+        fmt::print(stdout,
+                   "the replica count of app({}) is {}\n",
+                   escaped_app_name,
+                   resp.max_replica_count);
```

Comment:
It's better to output also in JSON, so we can parse easily in 3rd party tools.

Reply:
> It's better to output also in JSON, so we can parse easily in 3rd party tools.

OK, I'll add `-j` and `--json` for `get_replica_count`, for example: {"max_replica_count": 3}.

Reply:
You can use `::dsn::utils::table_printer` to wrap these code.

Reply:
> You can use `::dsn::utils::table_printer` to wrap these code.

OK.